pca/db/database_model.py

```diff
@@ -417,3 +419,38 @@ def save(self, *args, **kwargs):
         else:
             self.internal_ip_int = None  # IPv6 integer representations are too large to store in MongoDB currently
         super(ApplicationDoc, self).save(*args, **kwargs)
+
+class UserReportDoc(RootDoc):
+    """Provides schema for User Report doc.
+    customer - a customer ID that corresponds to a CustomerDoc
+    assessment - an assessment ID that corresponds to an AssessmentDoc
+    campaign - a campaign ID that corresponds to a CampaignDoc
+    first_report - the timestamp when the first user click ("report") was received during a phishing campaign
+    total_num_reports - the total number of user clicks ("reports") for a campaign
+    """
+    customer = fields.ReferenceField(CustomerDoc, required=True)
+    assessment = fields.ReferenceField(AssessmentDoc, required=True)
+    campaign = fields.ReferenceField(CampaignDoc, required=True)
+    first_report = fields.DateTimeField(blank=True)
+    total_num_reports = fields.IntegerField(blank=False)
```

Comment:
@JCantu248 - Sorry to go back to this, but is there any reason why this can't be:
```suggestion
    total_num_reports = fields.IntegerField(required=True)
```

Reply:
I set it that way because I wanted to account for having no reports generated, but then I have first_report set to true, so they are contradictory. I may want to make first_report false as well in the event that no user responds to a phishing email.